# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../gpm/strchr.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

__UCLIBC_SUSV3_LEGACY__

--- ./src/synaptics.c.orig	2010-04-21 12:02:48.000000000 -0400
+++ ./src/synaptics.c	2010-04-21 12:13:11.000000000 -0400
@@ -2134,9 +2134,9 @@
 	      break;
 
 	    case Flag_Param:
-	      if (index ("YyTt1", token [0])) {
+	      if (strchr ("YyTt1", token [0])) {
 		*(param_data [param].addr.flag_p) = 1;
-	      } else if (index ("NnFf0", token [0])) {
+	      } else if (strchr ("NnFf0", token [0])) {
 		*(param_data [param].addr.flag_p) = 0;
 	      } else {
 		gpm_report (GPM_PR_WARN,"Flag value (%s) for parameter %s is invalid",
